docs/esp8266/tutorial/network_basics.rst

```diff
@@ -1,14 +1,14 @@
 Network basics
 ==============
 
-The network module is used to configure the WiFi connection.  There are two WiFi
+The :class:`network.WLAN` class in the :mod:`network` module is used to configure the WiFi connection.  There are two WiFi
```

Comment:
maybe line break between "module" and "is used" to keep the lines somewhat the same length in this paragraph without needing to reflow the whole thing?

Reply:
I ended up reflowing this paragraph and adjusting a couple of others for long lines